src/index.js

```diff
@@ -1,10 +1,23 @@
 import React, { PureComponent } from 'react';
-import { Keyboard, Modal, Platform, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import {
+    Keyboard,
+    Modal,
+    Platform,
+    Text,
+    TextInput,
+    TouchableOpacity,
+    View,
+    Dimensions,
+} from 'react-native';
 import PropTypes from 'prop-types';
 import isEqual from 'lodash.isequal';
 import { Picker } from '@react-native-picker/picker';
 import { defaultStyles } from './styles';
 
+// Measuring the modal before rendering is not working reliably, so we need to hardcode the height
+// This height was tested thoroughly on several iPhone Models (from iPhone 8 to 14 Pro)
```

Comment:
tested with iPhone SE / Mini variants too?

also - does landscape mode work?

Reply:
yes, it was compatible, but i just made some improvements 👍 🚀

Reply:
hey, sorry for the late reply...

i think it's difficult to add tests here, because this is basically an iOS specific feature. With `jest`/`enzyme` we can only test in a mocked environment.

I added a dummy test, that reflects a test for this functionality, but this will not work in a web environnment